### ISSUE TITLE
nics.nicextraparams with spaces (ex: BONDING_OPTS) are faulty

### ISSUE THREAD

**Issue by Obihoernchen** [xcat2/xcat-core#6494]
Adding parameters with spaces like:
```nicextraparams.bond0=BONDING_OPTS="mode=active-backup miimon=100"```
fails because in `/install/postscripts/xcatlib.sh` the function: `function parse_nic_extra_params` splits the string at spaces.
See:  https://github.com/xcat2/xcat-core/blob/master/xCAT/postscripts/xcatlib.sh#L710-L737

Result:
```
# cat /etc/sysconfig/network-scripts/ifcfg-bond0
ONBOOT="yes"
USERCTL="no"
TYPE="Bond"
BONDING_MASTER="yes"
BONDING_OPTS=""mode=active-backup"
BOOTPROTO="static"
DHCLIENTARGS="-timeout 200"
BRIDGE="brbond0"
MTU="1500"
DEVICE="bond0"
NAME="bond0"
miimon="100""
```

Should be:
```
# cat /etc/sysconfig/network-scripts/ifcfg-bond0
ONBOOT="yes"
USERCTL="no"
TYPE="Bond"
BONDING_MASTER="yes"
BONDING_OPTS="mode=active-backup miimon=100"
BOOTPROTO="static"
DHCLIENTARGS="-timeout 200"
BRIDGE="brbond0"
MTU="1500"
DEVICE="bond0"
NAME="bond0"
```

Current behaviour is that `nic.nicextraparams` is splitted at every space but it shouldn't split if there are `"` because some parameters like `BONDING_OPTS` need spaces.

**Comment by gurevichmark** [comment]
It appears by the time the `BONDING_OPTS` are parsed by `xcatlib.sh` the quotes around the parameters are no longer there. In `nic` table the `BONDING_OPTS` are stored simply as `...,,"bond0!BONDING_OPTS=mode=active-backup miimon=100",...`

@Obihoernchen do you know if `/etc/sysconfig/network-scripts/ifcfg-bond0` can support multiple `BONDING_OPTS` entries like:
```
:
:
AUTOCONNECT_RETRIES=0
AUTOCONNECT_SLAVES=yes
BONDING_OPTS=mode=active-backup
BONDING_OPTS=miimon=100
```

If something like this is supported, then the following could be a workaround:
`chdef <node> nicextraparams.bond0="BONDING_OPTS=mode=active-backup BONDING_OPTS=miimon=100"`

**Comment by Obihoernchen** [comment]
I don't think so.

**Comment by Obihoernchen** [comment]
I just checked and it overwrites the first `BONDING_OPTS` entry. Therefore, this does not work.

**Comment by gurevichmark** [comment]
For unit test, verify something like this works:
`chdef <node> nicextraparams.bond0='BONDING_OPTS="mode=active-backup abc=100" MTU=6400 XYZ="4800" IOP="mode=1 phase=2"'`

**Comment by gurevichmark** [comment]
Fixed by #6599
